lms/static/certificates/sass/_print.scss

```diff
@@ -94,6 +94,12 @@
 // ------------------------------
 @media print {
 
+    html,
+    body {
+        height: 650px;
+        page-break-after: avoid;
```

Comment:
I vote we keep it "just in case", as this is the 'proper' way to do it. If you guys wanna lose it, I could be swayed.